### DISCUSSION TITLE
SSD-1B (distilled SDXL) compatibility

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/sdnext#2395] [General]
Found this here, was released not long ago: https://blog.segmind.com/introducing-segmind-ssd-1b/

Its for diffusers. Before I try anything, would this work out of the box with SD Next together with the refiner?

**Comment by Aptronymist** [comment]
We've already noticed this late last night, it will be addressed because with it being released just yesterday, we aren't set up to support that one just yet. As far as using a refiner with it... I have no idea.
